### PR TITLE
Programme trips select governorate

### PR DESCRIPTION
**Pull request by ntrncic**
Added TripLocation Model to trips
Changed the form where user must have a TtripLocation if the tavel type is programme monitoring.

### CODE REVIEW

File: EquiTrack/trips/forms.py [hunk 1]
```diff
@@ -161,6 +162,25 @@ def clean(self):
         return cleaned_data
 
 
+# class LinkedLocationForm(ModelForm):
+#     def clean(self):
+#         cleaned_data = super(LinkedLocationForm, self).clean()
+#         print 'test'
+
+
+class RequiredInlineFormSet(BaseInlineFormSet):
```
Comment (jamescw) [inline EquiTrack/trips/forms.py]: ok this good just change the name to RequiredLocationInlineFormset

**Comment by jamescw** [comment]
ok this is perfect, nice and clean, only thing we need is datamigration to migrate trip locations to this model instead of the old one, also I will help with the test for this

**Comment by jamescw** [comment]
@ntrncic just need a data migration and then good to merge

**Comment by jamescw** [comment]
@ntrncic good to merge when test passes

**Comment by ntrncic** [comment]
@jamescw ready to merge, finally!

**Comment by jamescw** [comment]
@ntrncic boom great work merging down

**Pull request merged**